extension/src/patch.ts

```diff
@@ -121,7 +135,16 @@ const shareWithProgress = (
       increment: 0,
       message: 'Initializing experiment...'
     })
-    await sendPostRequest(studioAccessToken, { ...base, type: 'start' })
+    const response = await sendPostRequest(studioAccessToken, {
+      ...base,
+      type: 'start'
+    })
+
+    if (response.status === 401) {
+      progress.report({ increment: 100, message: 'Access unauthorized' })
+      void showUnauthorized()
+      return Toast.delayProgressClosing()
```

Comment:
[F] Another option that I considered was to remove the token here as it has been shown to be invalid but I thought that would be too intrusive.